src/Then.php

```diff
@@ -13,9 +13,18 @@ interface Then
 {
     public function then(callable $callback): Then;
 
+    /**
+     * @param array<mixed, mixed> $values
+     */
     public function thenEach(array $values, callable $callback): Then;
 
+    /**
+     * @param bool $truth
+     */
     public function thenIf($truth, callable $yes, callable $no = null): Then;
 
+    /**
+     * @param bool $truth
```

Comment:
Should they be `mixed` instead?

Reply:
No, I think you're right that $truth should be a concrete bool - it was left open for potentially cleaner call signatures, but actually I'm not sure it makes a difference and otherwise should be stricter.

I will update the interface shortly.